### PR TITLE
[full-ci] ci: remove special PR-based drone actions from CI

### PR DESCRIPTION
**Pull request by JammingBen**
Remove special PR-based drone actions from CI to enable running pipelines again.

### CODE REVIEW

**Review by phil-davis — APPROVED** [review]
LGTM if CI is green

File: .drone.star [hunk 1]
```diff
@@ -667,10 +665,10 @@ ocisSpecificTestSuites = [
 # minio mc environment variables
 minio_mc_environment = {
     "CACHE_BUCKET": {
-        "from_secret": "cache_public_s3_bucket",
+        "from_secret": "cache_s3_bucket",
```
Comment (saw-jan) [inline .drone.star:668]: I think, `cache_public_s3_bucket`  can be left unchanged
```suggestion
        "from_secret": "cache_public_s3_bucket",
```

File: .drone.star [hunk 2]
```diff
@@ -2534,40 +2502,17 @@ def cacheOcis():
...
         "image": PLUGINS_S3,
         "pull": "if-not-exists",
         "settings": {
             "bucket": {
-                "from_secret": "cache_public_s3_bucket",
+                "from_secret": "cache_s3_bucket",
             },
             "endpoint": {
-                "from_secret": "cache_public_s3_server",
+                "from_secret": "cache_s3_server",
```
Comment (saw-jan) [inline .drone.star:2515]: `cache_public_s3_server` can be left unchanged
```suggestion
                "from_secret": "cache_public_s3_server",
```

File: .drone.star (hunk 1 above)
Comment (JammingBen) [reply .drone.star:668]: Are you sure? Because in the ocis repo these have changed: https://github.com/owncloud/ocis/pull/6714/files

Comment (saw-jan) [reply .drone.star:668]: Ahh, didnt notice that. But the secret is still available. @phil-davis is there any difference?

**Comment by JammingBen** [comment]
@xoxys @phil-davis @micbar The `cache-pnpm` step seems to be stuck, I assume there is still some issue with the cache setup: https://drone.owncloud.com/owncloud/web/37079/5/3 Do you have any idea what's missing?

File: .drone.star (hunk 1 above)
Comment (xoxys) [reply .drone.star:668]: It all has to match. If it is intended to upload files to the public cache (everyone will be able to publically read those files without auth from the cache server) `cache_public_s3_*` vars have to be used. If you want to use the private cache use `cache_s3_*`.

File: .drone.star (hunk 2 above)
Comment (xoxys) [reply .drone.star:2515]: Don't mix it. Use consistent secrets `cache_public_s3_*` _or_ `cache_s3_*`

File: .drone.star (hunk 1 above)
Comment (xoxys) [reply .drone.star:668]: The public cache is intended to be used for screenshots, logs etc. that needs to be added to PR comments. Again, files uploaded to the public cache will be readable for everyone, use it with caution.

**Comment by xoxys** [comment]
@JammingBen There are still occurrences of `cache_s3_endpoint` in the drone config. This needs to be changed, as the secret does not exist anymore. Example https://github.com/owncloud/web/blob/4bad4d78de25ecc9fa6b2dea5529d96c33356a63/.drone.star#L2915

File: .drone.star (hunk 1 above)
Comment (saw-jan) [reply .drone.star:668]: So, for e2e tracings `cache_public_s3_*` is best suited. So that we can view tracing without auth

Comment (saw-jan) [reply .drone.star:668]: > I am still using `cache_public_s3_bucket` in https://github.com/owncloud/client/pull/11015/files

we are using s3_cache to upload gui test logs which should be viewable without any auth. so `cache_public_s3_bucket` seems right there too

**Review by xoxys — APPROVED** [review]
Looks good now, if the CI is still failing feel free to ping me again

**Pull request merged**